client-rpc/src/client_rpc.rs

```diff
@@ -0,0 +1,376 @@
+use chain_core::init::coin::Coin;
+use chain_core::state::account::{StakedStateAddress, StakedStateOpAttributes};
+use chain_core::tx::data::address::ExtendedAddr;
+use chain_core::tx::data::attribute::TxAttributes;
+use chain_core::tx::data::input::TxoPointer;
+use chain_core::tx::data::output::TxOut;
+use client_common::{Error, ErrorKind, PublicKey};
+use client_network::network_ops::NetworkOpsClient;
+use jsonrpc_core::Result;
+use jsonrpc_derive::rpc;
+use secstr::SecUtf8;
+use serde::{Deserialize, Serialize};
+use std::str::FromStr;
+
+#[rpc]
+pub trait ClientRpc {
```

Comment:
Any particular reason for creating a new trait instead of adding these functions to existing `WalletRpc` trait?

Reply:
it's possible to make it one,
how about changing name to ClientRpc from WalletRpc.
because it does wallet and other tx stuff, more general name would be good.
what do you think?

Reply:
I'm fine with anything -- e.g. to rename WalletRpc to ClientRpc and move all functionality there. the distinction could be just done on the JSON-RPC method name level (call / prefix "wallet.<method name>" for value transfer ops, prefix "stake.<method name>" for network stake ops?)

Reply:
ok,
i'll merge to one.